datadog/resource_datadog_security_monitoring_rule.go

```diff
@@ -669,6 +893,27 @@ func extractFiltersFromRuleResponse(ruleResponse *datadogV2.SecurityMonitoringSt
 	return filters
 }
 
+func extractRuleCases(responseRulesCases []datadogV2.SecurityMonitoringRuleCase) []interface{} {
+	ruleCases := make([]interface{}, len(responseRulesCases))
```

Comment:
Nit to make the output type more explicit
```suggestion
func extractRuleCases(responseRulesCases []datadogV2.SecurityMonitoringRuleCase) []map[string]interface{} {
	ruleCases := make([]map[string]interface{}, len(responseRulesCases))
```